graphql-schema/proposal.graphql

```diff
@@ -93,7 +108,15 @@ input QueryProposalsInput {
   searchTerm: String
 }
 
+type ProposalVotesDistribution {
```

Comment:
i think it will make the api more useful if we return actual votes count here, and let ui layer compute the ratios